messages/lightning.preview.app.md

```diff
@@ -21,11 +21,23 @@ Use the appropriate topic to preview specific aspects of the development environ
 
 # flags.name.summary
 
-Description of a flag.
+Name of the Lightning Experience application to preview.
 
-# flags.name.description
+# flags.target-org.summary
 
-More information about a flag. Don't repeat the summary.
+Specify the org to preview.
```

Comment:
We should probably use some variation of [the verbiage](https://github.com/search?q=org%3Asalesforcecli%20%22Username%20or%20alias%20of%20the%20target%20org%22&type=code) that other CLI plugins use, for this description.